db-demo-product/README.md

```diff
@@ -0,0 +1,12 @@
+# database-demos
+
+[![CI Build](https://github.com/axonivy-market/db-demos/actions/workflows/ci.yml/badge.svg)](https://github.com/axonivy-market/db-demos/actions/workflows/ci.yml)
+
+Axon Ivy's database demo shows how to create, Read, Update and delete data tables. With the help of database activities, you can integrate existing databases perfectly.
```

Comment:
Almost perfect: I'd say `to create, read, update and delete data from database tables.`